aips/spark/__init__.py

```diff
@@ -1,8 +1,21 @@
 from pyspark.sql import SparkSession
+
 from aips.environment import AIPS_ZK_HOST
+from engines.solr.SolrCollection import SolrCollection
+
+from engines.opensearch.OpenSearchCollection import OpenSearchCollection
+from engines.opensearch.config import OPENSEARCH_URL
+
 
 def create_view_from_collection(collection, view_name, spark=None):
     if not spark:
         spark = SparkSession.builder.appName("AIPS").getOrCreate()
-    opts = {"zkhost": AIPS_ZK_HOST, "collection": collection.name}    
-    spark.read.format("solr").options(**opts).load().createOrReplaceTempView(view_name)
\ No newline at end of file
+    if isinstance(collection, SolrCollection):
```

Comment:
Let's add a "get_engine_name" function or similar to the Collection class. This removes the need to import all the Collections to do `isinstance` checks in various places throughout the code base.

Reply:
I've made this change and implemented it into a switch/match statement which wasn't possible when types are being checked. So this reads better now.